src/elements/signin-dialog.html

```diff
@@ -0,0 +1,112 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/iron-overlay-behavior/iron-overlay-behavior.html">
+<link rel="import" href="../mixins/redux-mixin.html">
+<link rel="import" href="../elements/shared-styles.html">
+<link rel="import" href="../elements/hoverboard-icons.html">
+
+<dom-module id="signin-dialog">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+    <style>
+        :host {
+            margin: 0 auto;
+            display: block;
+            padding: 24px 32px;
+            background: #fff;
+            box-shadow: var(--box-shadow);
+            
+        }
+
+        .dialog-content {
+            margin: 0 auto;
+        }
+
+        .provider-label {
+            margin-left: 24px;
+            line-height: 20px;
+        }
+
+        paper-button {
+            height: 50px;
+            margin: 8px 0;
+            color: var(--secondary-text-color);
+            display: block;
+        }
+
+        .icon-google {
+            padding-top: 3px;
+            --iron-icon-height: 25px;
+            --iron-icon-width: 25px;
+        }
+
+        .icon-twitter {
+            color: var(--twitter-color);
+        }
+
+        .icon-facebook {
+            color: var(--facebook-color);
+        }
+      
+
+        @media (min-width: 640px) {  
+
+         
+        }
+
+     
+    </style>
+
+    <div class="dialog-content" layout vertical start>
+        {% for provider in signInProviders.providersData %}
+            <paper-button 
+                on-tap="{$ provider.name $}SignIn"
+                provider-url="{$ provider.url $}"
+                ga-on="click"
+                ga-event-category="attendees"
+                ga-event-action="sign-in"
+                ga-event-label="signIn dialog - {$ provider.name $}"
+                layout horizontal center-center>
+                <iron-icon class="icon-{$ provider.name $}" icon="hoverboard:{$ provider.name $}"></iron-icon>
+                <span class="provider-label">{$ provider.label $}</span>
+            </paper-button>
+        {% endfor %}
+    </div>
+    
+    
+  </template>
+
+  <script>
+
+    class SigninDialog extends ReduxMixin(Polymer.mixinBehaviors([Polymer.IronOverlayBehavior], Polymer.Element)) {
+
+    static get is() {
+            return 'signin-dialog'
+        }
+
+        static get properties() {
+            return {                
+                subscribed: {
+                    type: Boolean,
+                    statePath: 'subscribed'
+                }
+            }
+        }  
+
+        googleSignIn() {
```

Comment:
it's would be better create one generic function and get provider value from attributes